Completed Newsweek clone

DennisOsido opened the pull request:
Live Preview: https://rawcdn.githack.com/DennisOsido/Newsweek-Clone/9a80820c7def71b396d5ddb5876e4156a6302c53/index.html

## EDIT after the third review:
-  Removed Horizontal Scrollbar and Added Media Queries for Debate Section

## EDIT after the second review:
- Added Media Queries and Styles for Responsiveness with emphasis for the following screen sizes: 320px, 768px, and 1024px as recommended by TSE

## EDIT after the first review:
- Removed all inline styling
- "The Debate", "Newsletter" and "Subscribe" sections created piece by piece
- Fixed Navbar alignment and Footer Section
- Fixed "Editor's Pick" section

## In this solo project I have:
- Created a replica of Newsweek.com using Bootstrap classes wherever possible.

## Steps:
- Created a Header
- Created a Navbar
- Created 5 Sections
- Created a Footer section
- Added Bootstrap classes

## Requirements that were completed:
- Bootstrap classes were used wherever possible
- Made sure that page layout looks exactly like the source page
- Made sure that page layout does not break on smaller screens

DennisOsido committed:
Added Stickler and Linter

DennisOsido committed:
Added Bootstrap Boilerplate

DennisOsido committed:
Added Header and CSS Reset

DennisOsido committed:
Added Navbar

DennisOsido committed:
Added Section 1

DennisOsido committed:
Added Readme

DennisOsido committed:
Added Images and Styles to Section 1

DennisOsido committed:
Completed Section 1

DennisOsido committed:
Added Media Query to Navbar

DennisOsido committed:
Added Media Query to Header for Tablet and Mobile

DennisOsido committed:
Added Bootstrap styles to column layout for Tablet and Mobile

DennisOsido committed:
Added Section 2

DennisOsido committed:
Fixed HTML Validation Errors

DennisOsido committed:
Added Section 3

DennisOsido committed:
Added Section 4

DennisOsido committed:
Added Section 5

DennisOsido committed:
Added 1st Footer

DennisOsido committed:
Added 2nd Footer

DennisOsido committed:
Added 3rd Footer

DennisOsido committed:
Updated Readme

pelzolga123 reviewed (changes requested):
Project requires changes.

pelzolga123 commented on index.html:
```diff
@@ -0,0 +1,1788 @@
+          </div>
+          <div>
+            <a class="header-left-1" href="#">
+              <div style="display: flex;">
```
No inline styling, all styles should be inside styles.css file.

pelzolga123 commented:
## Hi Dennis! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
### Your project looks :+1: great!
Here are the things you need to change before this project can be approved:
* [ ] This elements should be created piece by piece not an image:
 ![image](https://user-images.githubusercontent.com/20416431/76708391-17c98d80-66ff-11ea-95af-719742dbab58.png)

![image](https://user-images.githubusercontent.com/20416431/76708400-244de600-66ff-11ea-9c81-6c6f3b98ae6d.png)

![image](https://user-images.githubusercontent.com/20416431/76708425-5cedbf80-66ff-11ea-85e3-54af88f0cff0.png)

* [ ] Try to find font, similar to original website, right now it looks very different. Also try to replicate similar font weight and size to make a better Newsweek clone. 

### Tablet version
* [ ] Navbar should look more like original, all elements should be aligned properly:
![image](https://user-images.githubusercontent.com/20416431/76708498-f5843f80-66ff-11ea-94a7-107a86308c5b.png)
Original:
![image](https://user-images.githubusercontent.com/20416431/76708516-0cc32d00-6700-11ea-86cb-12dd14bd6c23.png)
 * [ ] Editor's pick section should change like original:
![image](https://user-images.githubusercontent.com/20416431/76708575-6fb4c400-6700-11ea-86dc-25918d4aed55.png)
 
Original:
![image](https://user-images.githubusercontent.com/20416431/76708581-8ce99280-6700-11ea-85cd-e8b1bd6f14f8.png)
 * [ ] Footer should look more like original:
![image](https://user-images.githubusercontent.com/20416431/76708594-bf938b00-6700-11ea-87b7-d2799ec2ff68.png)

Original:
![image](https://user-images.githubusercontent.com/20416431/76708598-ce7a3d80-6700-11ea-9304-4751599185dc.png)

### Mobile version
* [ ] It has white line on the right side, that should be removed:
![image](https://user-images.githubusercontent.com/20416431/76708646-2d3fb700-6701-11ea-95f5-d4b8e76fbbcf.png)

Original:
![image](https://user-images.githubusercontent.com/20416431/76708656-46486800-6701-11ea-8d03-d8324060548e.png)

* [ ] Sections should be displayed in next order:
* Top Story
* Featured Stories
* Opinion
* The Debate
* More Stories
* Culture & Travel
* Latest News


### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

DennisOsido committed:
Created Debate section piece by piece

DennisOsido committed:
Created Newsletter section piece by piece

DennisOsido committed:
Created Subscribe  section piece by piece

DennisOsido committed:
Fixed Navbar alignment and Footer Section

DennisOsido committed:
Updated breakpoint of Editors Pick section

DennisOsido commented:
## After the first review:
- Removed all inline styling
- "The Debate", "Newsletter" and "Subscribe" sections created piece by piece
- Fixed Navbar alignment and Footer Section
- Fixed "Editor's Pick" section

nikasulo reviewed (changes requested):
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback! :+1:

nikasulo reviewed (changes requested):
# Great work putting this together @DennisOsido :+1: 
Please consider my comment below towards tying this all together, make the necessary changes and request another review. I wish you the best!

#### Responsiveness
Your project could use a little more responsiveness, please look a little more closely at the sizes of elements, padding of sections, margins, and general element positionings. These are very key as they will affect the look, feel, and more importantly the usability of your sites. Please consider(at least) the following screen sizes: 320px, 768px, and 1024px, use your browser developer tools.

Try to make each section look as close to the original site as possiible

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

DennisOsido committed:
Added Media Queries and Styles for Responsiveness

DennisOsido commented:
## After the second review:

- Added Media Queries and Styles for Responsiveness with emphasis for the following screen sizes: 320px, 768px, and 1024px as recommended by TSE

rOluochKe reviewed (changes requested):
Hello Team!,
### You have done a good job so far and for also implementing changes from previous reviews :clap: However, you have a few changes to make before your project could be approved.
- Remove the horizontal scroll bar on mobile devices, the width needs to fit well
- Also, in the debate section, redesign to fit well on medium and small screens
![medium screen](https://user-images.githubusercontent.com/8114528/76843947-9df3ea00-684d-11ea-8bf3-c67cfabb20e4.PNG)
### Make these changes and resubmit for another code review. Good luck.
- Happy coding!

DennisOsido committed:
Removed Horizontal Scrollbar and Added Media Queries for Debate Section

DennisOsido commented:
## After the third review:
-  Removed Horizontal Scrollbar and Added Media Queries for Debate Section

rOluochKe commented:
Hello Team!  
### Good job and for also making changes from previous reviews. You project looks good and I therefore, approve of it :boom: :100:
### you can merge and submit a project completion form from your dashboard. Good luck with the next project.
- Happy coding.
